src/lib/components/viewer/ReadingToolbar.svelte

```diff
@@ -95,19 +103,23 @@ Assumes it's a child of a ViewerContext
       <Dropdown position="bottom-start">
         <SidebarItem slot="anchor">
           <svelte:component this={icons[$mode]} slot="start" />
-          {Array.from(readModes ?? []).find(([value]) => value === $mode)?.[1]}
+          {Array.from(readModeDropdownItems ?? []).find(
+            ([value]) => value === $mode,
+          )?.[1]}
           <ChevronDown12 slot="end" />
         </SidebarItem>
         <Menu slot="default" let:close>
-          {#each readModes.entries() as [value, name]}
-            <MenuItem
-              selected={$mode === value}
-              href={getViewerHref({ document, mode: value, embed })}
-              on:click={close}
-            >
-              <svelte:component this={icons[value]} slot="icon" />
-              {name}
-            </MenuItem>
+          {#each readModeDropdownItems.entries() as [value, name]}
+            {#if value !== "search"}
+              <MenuItem
+                selected={$mode === value}
+                href={getViewerHref({ document, mode: value, embed })}
```

Comment:
This is losing the search query when we switch modes. I think we want to keep it.

Reply:
I think we also want to keep search mode as a dropdown option, so I can switch back to results I've already seen.

Reply:
Good catch on the missing query. I think you're right on returning to results here—it's more awkward on mobile to pop open and resubmit the search form. On desktop, I think we're ok keeping results under a search form submission.